src/dlgTriggerEditor.cpp

```diff
@@ -2530,7 +2562,7 @@ void dlgTriggerEditor::addTrigger( bool isFolder )
     if( pParent ) pParent->setExpanded( true );
     mpTriggersMainArea->lineEdit_trigger_name->clear();
     mpTriggersMainArea->perlSlashGOption->setChecked( false );
-    mpSourceEditor->clear();
+    mpSourceEditorEdbeeDocument->setText("");
```

Comment:
It doesn't have a `clear()` method to remove the contents?
Please avoid using raw string literals {we will only have to go through later and mark whether they are part of the UI and need marking as something that has to be translated (or not) for Mudlet 4.0...} - I'd suggest the following:

    mpSourceEditorEdbeeDocument->setText(QString());